lib/tasks/purge.rake

```diff
@@ -0,0 +1,8 @@
+namespace :purge do
```

Comment:
Same as above, we should failsafe to make sure it never runs where it shouldn't

Reply:
[Addressed here.](https://github.com/simpledotorg/simple-server/pull/786/commits/8c30c3878505e0a30003dfc87ab64c870b2bc448)